image/autodetect.go

```diff
@@ -61,8 +61,6 @@ func Autodetect(path string) (string, error) {
 
 	switch mimeType {
 	case "application/x-gzip":
-		return TypeImage, nil
-
 	case "application/octet-stream":
```

Comment:
@wking It is no problem, I have verified it.

In generally, Go's `switch` supports style `case <con1>,<con2>:` and that I have submitted.